openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummaryCustomTooltip.component.tsx

```diff
@@ -28,21 +36,57 @@ const TestSummaryCustomTooltip = (
   return (
     <Card
       title={
-        <Typography.Title level={5}>{payload[0].payload.name}</Typography.Title>
+        <Typography.Title level={5}>
+          {formatDateTime(payload[0].payload.name)}
+        </Typography.Title>
       }>
-      <ul data-testid="test-summary-tool-tip-container">
-        {data.map(([key, value]) => (
-          <li
-            className="d-flex items-center justify-between gap-6 p-b-xss text-sm"
-            key={`item-${key}`}>
-            <span className="flex items-center text-grey-muted">
-              {startCase(key)}
-            </span>
-            <span className="font-medium" data-testid={key}>
-              {value}
-            </span>
-          </li>
-        ))}
+      <ul data-testid="test-summary-tooltip-container">
```

Comment:
Can we use AntD List component here?

Reply:
I will address this comment separately as I need to make similar changes in one more component. Thanks